src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -444,7 +444,7 @@
                 return;
             }
 
-            const modal = this.container.closest('.modal');
+            const modalDialog = this.container.closest('.modal-dialog');
```

Comment:
Clicking inside `.modal`, but outside `.modal-dialog` is treated by Bootstrap as clicking outside modal, which may close it in some cases. Thus we render the dropdown popover inside `.modal-dialog`.